pkg/server/formsapi/handlers/form/create.go

```diff
@@ -56,6 +62,47 @@ func (h *Handler) Create() http.HandlerFunc {
 	}
 }
 
+func validateRecipientFormParent(
```

Comment:
shouldn't this also live in validation/recipient.go?

Reply:
@ludydoo

Reply:
@internetti the problem with this one is that it requires an access to the database. All other validation happening in validation/recipient.go is "static" (does not need any service call, etc.). Do you think it would be cleaner if it would be in the validation/recipient.go ?

Reply:
This piece of code should probably live in some kind of controller/validation webhook I think. But we don't have that layer yet !

Reply:
What about passing the store to `ValidateForm`?

Reply:
@neb42 that would work, though it would require all tests to inject the store as well. Imho I really think this should be on a controller layer of some kind.